assets/js/plugin-check-admin.js

```diff
@@ -35,6 +35,31 @@
 	canRunChecks();
 	pluginsList.addEventListener( 'change', canRunChecks );
 
+	/**
+	 * Saves the user settings.
+	 *
+	 * @since n.e.x.t
```

Comment:
Probably not needed. This function is not exposed to global scope or anything.

Reply:
Thanks for the review. I've opened a separate issue #305, to address this matter.